pkgs/tools/virtualization/guestfs-tools/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, fetchurl
+, bison
+, cdrkit
+, cpio
+, flex
+, getopt
+, hivex
+, jansson
+, libguestfs
+, libvirt
+, libxml2
+, makeWrapper
+, ncurses
+, ocamlPackages
+, pcre2
+, perlPackages
+, pkg-config
+, qemu
+, xz
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guestfs-tools";
+  version = "1.48.2";
+
+  nativeBuildInputs = [
+    bison
+    cdrkit
+    cpio
+    flex
+    getopt
+    makeWrapper
+    pkg-config
+    qemu
```

Comment:
Is `qemu` really needed during the build?

Reply:
Yes, otherwise:

```
qemu-img create -f raw blank-disk-1s.raw 512
qemu-img create -f qcow2 -o preallocation=metadata blank-disk-1s.qcow2 512
qemu-img create -f raw blank-disk-1K.raw 1K
qemu-img create -f qcow2 -o preallocation=metadata blank-disk-1K.qcow2 1K
qemu-img create -f raw blank-disk-1M.raw 1M
qemu-img create -f qcow2 -o preallocation=metadata blank-disk-1M.qcow2 1M
/nix/store/2ispfz80kmwrsvwndxkxs56irn86h43p-bash-5.1-p16/bin/bash: line 1: qemu-img: command not found
make[3]: *** [Makefile:723: blank-disk-1s.raw] Error 127
make[3]: *** Waiting for unfinished jobs....
/nix/store/2ispfz80kmwrsvwndxkxs56irn86h43p-bash-5.1-p16/bin/bash: line 1: qemu-img: command not found
make[3]: *** [Makefile:726: blank-disk-1s.qcow2] Error 127
/nix/store/2ispfz80kmwrsvwndxkxs56irn86h43p-bash-5.1-p16/bin/bash: line 1: qemu-img: command not found
/nix/store/2ispfz80kmwrsvwndxkxs56irn86h43p-bash-5.1-p16/bin/bash: line 1: qemu-img: command not found
make[3]: *** [Makefile:732: blank-disk-1K.qcow2] Error 127
make[3]: *** [Makefile:729: blank-disk-1K.raw] Error 127
/nix/store/2ispfz80kmwrsvwndxkxs56irn86h43p-bash-5.1-p16/bin/bash: line 1: qemu-img: command not found
/nix/store/2ispfz80kmwrsvwndxkxs56irn86h43p-bash-5.1-p16/bin/bash: line 1: qemu-img: command not found
make[3]: *** [Makefile:735: blank-disk-1M.raw] Error 127
make[3]: *** [Makefile:738: blank-disk-1M.qcow2] Error 127
```